app/views/users/show.html.erb

```diff
@@ -24,15 +24,29 @@
     }
     <% end %>
   </style>
-  <div class="user-profile-header" style="<%= user_colors_style(@user) %>" itemscope itemtype="http://schema.org/Person" itemprop="mainEntity">
-    <meta itemprop="url" content="https://dev.to/<%= @user.username %>">
+
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Person",
+    "mainEntityOfPage": "True",
+    "url": "https://dev.to/<%= @user.username %>",
+    "image": {
+      "@type": "ImageObject"
+    },
+    "name": "<%= @user.name %>",
+    "description": "<%= @user.summary.presence || ["404 bio not found"].sample %>"
+  }
+</script>
+
+<div class="user-profile-header" style="<%= user_colors_style(@user) %>">
     <div class="user-profile-header-container">
       <div class="profile-pic-wrapper">
-        <img class="profile-pic" src="<%= ProfileImage.new(@user).get(width: 320) %>" itemprop="image" alt="<%= @user.username %> profile" style="border-color:<%= user_colors(@user)[:bg] %>;background:<%= user_colors(@user)[:bg] %>" />
+        <img class="profile-pic" src="<%= ProfileImage.new(@user).get(width: 320) %>" alt="<%= @user.username %> profile" style="border-color:<%= user_colors(@user)[:bg] %>;background:<%= user_colors(@user)[:bg] %>" />
       </div>
       <div class="profile-details">
         <h1>
-          <span itemprop="name"><%= @user.name %><%= render "shared/pro_checkmark", style: nil, width: nil %></span>
```

Comment:
Looks like this deletion deleted a little too much. The `span` is no longer needed, but the content within should stay (You can see on the profile page, the name is no longer there as it should be.